tensorflow_datasets/core/registered.py

```diff
@@ -106,6 +107,14 @@
     version=r"[0-9]+\.[0-9]+\.[0-9]+",
 ))
 
+_skip_reg = True
+
+@contextmanager
+def skip_registeration():
```

Comment:
Please also add a unittest inside `registered_test.py` directly testing the feature.

Reply:
Okay